Subject: [PATCH] Fix crash when completing an Ultimate Rampage dungeon
---
Index: src/dr_floor/FloorEndingGui.hx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/dr_floor/FloorEndingGui.hx b/src/dr_floor/FloorEndingGui.hx
--- a/src/dr_floor/FloorEndingGui.hx	(revision cdfedcdf44118a27084d749501306968e0c7902a)
+++ b/src/dr_floor/FloorEndingGui.hx
@@ -208,7 +208,7 @@
       
       public function dungeonVictory() 
       {
-         if(mScreensRoot != null)
+         if(mScreensRoot != null && mVictoryRenderer != null)
          {
             displayVictory();
             checkToAddRootToScene();
